meinberlin/apps/dashboard2/contents.py

```diff
@@ -1,5 +1,27 @@
 class DashboardContents:
-    _registry = {}
+    _registry = {'project': {}, 'module': {}}
+
+    def __getitem__(self, identifier):
+        component = self._registry['project'].get(identifier, None)
+        if not component:
+            component = self._registry['module'].get(identifier)
+        return component
+
+    def __contains__(self, identifier):
+        return (identifier in self._registry['project'] or
+                identifier in self._registry['module'])
```

Comment:
I don't like to treat these two dictionaries as one. I would prefer to have `get_project_component(self, identifier)` and  `get_module_component(self, identifier)` instead.